packages/block-library/src/calendar/index.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Server-side rendering of the `core/calendar` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/calendar` block on server.
+ *
+ * @param array $attributes The block attributes.
+ *
+ * @return string Returns the block content.
+ */
+function render_block_core_calendar( $attributes ) {
+	global $monthnum, $year;
+	if ( isset( $attributes['monthNum'] ) ) {
+		$monthnum = $attributes['monthNum'];
+	}
+
+	if ( isset( $attributes['year'] ) ) {
+		$year = $attributes['year'];
+	}
+	$custom_class_name = empty( $attributes['className'] ) ? '' : ' ' . $attributes['className'];
+	$align_class_name  = empty( $attributes['align'] ) ? '' : " align{$attributes['align']}";
+
+	return sprintf(
+		'<div class="%1$s">%2$s</div>',
+		esc_attr( 'wp-block-calendar' . $custom_class_name . $align_class_name ),
```

Comment:
nit: Don't need to `esc_attr()` here since we know that `$custom_class_name` and `$align_class_name` are safe values.

Reply:
Isn't it possible that the user may apply a manual code change to something unsafe on the className attribute?

Reply:
Oops, you're totally right! 😊